arch/lkl/lib/string.c

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: GPL-2.0
+
+#include <linux/string.h>
+#include <linux/export.h>
+
+#undef memcpy
+#undef memset
+#undef memmove
+
+__visible void *memcpy(void *dest, const void *src, size_t count)
+{
+	return __memcpy(dest, src, count);
+}
+EXPORT_SYMBOL(memcpy);
+
+
```

Comment:
nit: please drop the double newlines here and below

Reply:
Right. I have no idea why I (accidentally) added two blank lines here... Thanks for pointing this out, I've updated accordingly.